api/src/main/java/com/thesurvey/api/dto/request/AnsweredQuestionRequestDto.java

```diff
@@ -9,13 +10,16 @@
 import lombok.Builder;
 import lombok.Getter;
 
+// FIXME: this should be fixed since answers should only have one of 4 question types
```

Comment:
Could you provide further explanation on the part that needs to be fixed??

Reply:
Okay then I'll make submitting answers in a parameter-form

Reply:
No problem on feature but only in swagger document, it shows all 4 answers in one question bank.

Reply:
I got it. Thank you for the explanation!